R/generate_dist.R

```diff
@@ -515,21 +515,11 @@
 .parseUnifFormula <- function(formula, dtSim, n, envir) {
   range <- .splitFormula(formula)
 
-  if (length(range) != 2) {
```

Comment:
Why the deletion?

Reply:
That is so strange, I did not intend to do that. I will add back.